tests/jobs/scala/build.sbt

```diff
@@ -12,7 +12,7 @@ lazy val root = (project in file("."))
       "org.apache.kafka" % "kafka_2.11" % "0.10.0.1",
       "com.github.scopt" %% "scopt" % "3.7.0",
       "com.datastax.spark" %% "spark-cassandra-connector" % "2.0.7",
-      "com.airbnb" % "kafka-statsd-metrics2" % "0.5.3" from "http://downloads.mesosphere.com/kafka/assets/kafka-statsd-metrics2-0.5.3.jar",
+      "com.airbnb" % "kafka-statsd-metrics2" % "0.5.2+issue28" from "https://infinity-artifacts.s3.amazonaws.com/scale-tests/kafka-statsd-metrics2-0.5.2+issue28-20180612-8c1e3c4f3fa83.jar",
```

Comment:
Not that it's critical, but we could consider moving this `jar` to a permanent `assets` folder.